rezerwacje-app/build.gradle

```diff
@@ -23,12 +23,14 @@ repositories {
 }
 
 dependencies {
-	compile 'org.postgresql:postgresql'
+	runtimeOnly 'org.postgresql:postgresql'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	compileOnly 'org.projectlombok:lombok'
 	developmentOnly 'org.springframework.boot:spring-boot-devtools'
 	annotationProcessor 'org.projectlombok:lombok'
+
+	testCompile group: 'com.h2database', name: 'h2', version: '1.3.148'
```

Comment:
This is a good idea to add versioning in the dependencies for compatibility. What about adding versions for others? But I would keep it in one style either "group:name:version" or separate as you did in h2 for consistency. It may be good idea to add versioning (tag) for postgres in docker-compose as well.

Reply:
Normally you can't add a dependency in gradle without specifying which version to use. It's only doable, because of the dependency-management plugin, so the versions won't change as long as we don't change spring boot plugin version.
I'm going to fix the style inconsistency.